configured webpack to create both bundles etc - configured for SSR

matgbx opened the pull request:
https://trello.com/c/i9OeYNiP/25-set-up-component-for-proxy-split-webpack-to-serve-both-bundles-etc

matgbx committed:
configured webpack and added route to server for server-bundle

matgbx committed:
removed unnecessary comments from node server

matgbx committed:
renamed as neccessary

matgbx committed:
update index.html to render bundle once received from client

matgbx committed:
mnior changes to avoid 'window is undefined' error on server

matgbx committed:
typo fix in index.html

matgbx committed:
Bug fixes, and optimized for SSR

lee-philip-n commented on webpack.prod.config.js:
```diff
@@ -1,44 +1,44 @@
-const path = require('path');
-const Dotenv = require('dotenv-webpack');
+// const path = require('path');
```
you could probably delete this file since everything is commented out

lee-philip-n reviewed (approved):
just one suggestion, no issues though, good job.

matgbx committed:
**Bug fixed - was displaying incorrect booking numbers on client due to redis caching
**Set up docker containers for service, redis and mongo
**seeded docker mongodb

matgbx committed:
configured component for deployed through docker

matgbx committed:
updated redis and mongo urls to accomodate local and deployed use

matgbx committed:
refactored client to use restaurant name for GET requests instead of id number

matgbx committed:
refactors to work with other proxies

matgbx committed:
set up artillery file for testing MongoDB
